trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative './decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.strip([0..9])
```

Comment:
* Kindly fix this error I get when I try to test your trimmer decorator class, I noticed that when I remove the parenthesis in front of the `strip` method the class works as expected 
![image](https://user-images.githubusercontent.com/52497246/199400704-c9e2406d-b6a9-415f-978b-3c969abc3223.png)
![image](https://user-images.githubusercontent.com/52497246/199400983-98893ac8-edf1-4d2b-a7b8-8e6899842260.png)